Add schema for Hadolint Dockerfile linter and Container Structure tests

brpaz opened the pull request:
This PR adds the schema for [hadolint](https://github.com/hadolint/hadolint), A Dockerfile linter

madskristensen commented:
Thanks. If you want this in the catalog API, please add it to `/src/api/json/catalog.json`

brpaz commented:
Thanks. I have added to the catalog. 
I have also added [Container Structure test](https://github.com/GoogleContainerTools/container-structure-test) schema.

madskristensen commented:
Thanks
